build.gradle

```diff
@@ -61,25 +61,33 @@ final depVersions = [
         springBoot         : ext.springBoot,
         commercetoolsSdkJvm: '1.27.0',
         paypalPlusSdk      : '1.14.0',
+        guava              : '23.0',
+        apacheCollections  : '4.0',
+
+        // tests
         assertJ            : '3.8.0',
-        guava              : '23.0'
+        systemRules        : '1.17.0'
 ]
 
 dependencies {
-    compile("org.springframework.boot:spring-boot-starter-web:${depVersions.springBoot}")
+    compile "org.springframework.boot:spring-boot-starter-web:${depVersions.springBoot}"
```

Comment:
not in this case: looks like Spring Boot plugin does not recognize properly this dependencies key-words and does not build proper fat jar (`:bootRepackage` task)

Reply:
there is some weird workaround, but honestly i don't like it so far:
https://github.com/spring-projects/spring-boot/issues/9143

Reply:
but you don't use the Java-library plugin. So according to the issue you linked it shouldn't be a problem for you? right?

Reply:
it is still problem - i tried twice (first time 2 weeks ago, second time - after your comment 2 days ago)

Reply:
Actually, that first time 2 weeks made me insane to find the problem: everything locally worked fine, the tests are running, but running service for the jar doesn't work.
I was lucky enough to figure out this before application releasing, otherwise it could be quite a hard bug on production and hard to find in the docker image itself